static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -175,7 +175,7 @@ define([
         var body;
 
         rules = rules || defaultRules;
-        body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        body = (rules.bodySelector && document.querySelector(rules.bodySelector)) || document;
```

Comment:
Thanks! Yeah, so I had a quick chat with @jbreckmckye and he pointed me towards the commercial-feature-policies so I added a new policy for the minute and turned off all the money makers :dancer: (as the plan is for this to be sponsored so we need dfp, but not any of the in-article modules). Thanks for your help!